Core/src/org/sleuthkit/autopsy/texttranslation/TranslationOptionsPanel.java

```diff
@@ -27,14 +27,12 @@
 import javax.swing.JLabel;
 import org.openide.util.NbBundle.Messages;
 import org.sleuthkit.autopsy.core.UserPreferences;
-import org.sleuthkit.autopsy.texttranslation.NoServiceProviderException;
-import org.sleuthkit.autopsy.texttranslation.TextTranslationService;
 import org.sleuthkit.autopsy.coreutils.Logger;
 
 /**
  * Options panel to display translation options
  */
-public class TranslationOptionsPanel extends javax.swing.JPanel {
+final class TranslationOptionsPanel extends javax.swing.JPanel {
```

Comment:
constructor now public in non-public class

Reply:
Thanks, will be fixed in next commit.